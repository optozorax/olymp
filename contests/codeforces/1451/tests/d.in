5
2 1
5 2
10 3
25 4
15441 33
