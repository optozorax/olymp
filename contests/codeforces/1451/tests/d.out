Utkarsh
Ashish
Utkarsh
Utkarsh
Ashish
